client/src/pages/home/home.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import logo from '../../logo.svg';
+import './home.css'; //import css for this page
+function Home() {   {/* This page is the exact same page as the original example */}
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Do not do this.